kind-1.24.yaml

```diff
@@ -2,4 +2,18 @@ kind: Cluster
 apiVersion: kind.x-k8s.io/v1alpha4
 nodes:
   - role: control-plane
-    image: kindest/node:v1.24.4@sha256:adfaebada924a26c2c9308edd53c6e33b3d4e453782c0063dc0028bdebaddf98
+    image: kindest/node:v1.24.12@sha256:1e12918b8bc3d4253bc08f640a231bb0d3b2c5a9b28aa3f2ca1aee93e1e8db16
+  kubeadmConfigPatches:
```

Comment:
Are these new config options required because of the upgrade from `v1.24.4` to `v1.24.12`?

Reply:
It seems we forget to add those options to 1.24. They should be there from the beginning